cfgov/jinja2/v1/index.html

```diff
@@ -13,64 +13,500 @@
+                                <ul class="list list__links">
+                                    <li class="list_item">
+                                        <a class="list_link"
+                                           href="#">
+                                            Learn more about credit cards, mortgages, and other financial products
+                                        </a>
+                                    </li>
+                                    <li class="list_item">
+                                        <a class="list_link"
+                                           href="#">
+                                            Find answers to common questions
+                                        </a>
+                                    </li>
+                                </ul>
+                            </div>
+                        </div>
+                    </section>
+                    <section class="content-l_col
+                                    content-l_col__home">
+                        <div class="m-info-unit
+                                    m-info-unit__home">
+                            <div class="m-info-unit_image
+                                        m-info-unit_image__square"
+                                 style="background-image: url(/static/img/home-goals_150.png);">
+                            </div>
+
+                            <div class="m-info-unit_content">
+                                <h2>Reach your financial goals
+                                </h2>
+                                <p>
+                                    For the big financial decisions in life, we offer free tools and resources to help you reach your goals.
+                                </p>
+                                <ul class="list list__links">
+                                    <li class="list_item">
+                                        <a class="list_link"
+                                           href="#">
+                                            Owning a Home
+                                        </a>
+                                    </li>
+                                    <li class="list_item">
+                                        <a class="list_link"
+                                           href="#">
+                                            Paying for College
+                                        </a>
+                                    </li>
+                                    <li class="list_item">
+                                        <a class="list_link"
+                                           href="#">
+                                            Planning for Retirement
+                                        </a>
+                                    </li>
+                                </ul>
+                            </div>
+                        </div>
+                    </section>
+                    {#
```

Comment:
Do you need this section and below? Can you add a TODO of why it's commented out?